hazelcast/src/main/java/com/hazelcast/internal/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -798,7 +803,7 @@ private boolean updatePartitionsAndFinalizeMigrations(InternalPartition[] partit
         // Manually trigger partition stamp calculation.
```

Comment:
Minor (for better understanding):
Possibly extend or update the description, since after the update there is no direct `updateStamp()` method call.

Reply:
updated in b222509